libraries/chain/db_maint.cpp

```diff
@@ -1062,6 +1062,23 @@ void process_hf_868_890( database& db, bool skip_check_call_orders )
    } // for each market issued asset
 }
 
+/****
+ * @brief a one-time data process to assign default taker fees for BSIP81
+ */
```

Comment:
below @brief consider more explanation

```
// everyone pays the taker fee - raw market fee - when placing a limit order
// later the maker is reimbursed through vesting balances
```
then in code below instead of the comment
```
// for each asset
```
be more descriptive
```
// get a list of all assets in the database by index
// iterate through the assets and set default fee fee to the taker amount
```